dev-libs/libfilezilla/libfilezilla-0.6.1.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="C++ library offering some basic functionality for platform-independent programs"
+DESCRIPTION="Some of the highlights include:
+
+- A typesafe, multi-threaded event system that's very simple to use yet \
+extremely efficient
+- Timers for periodic events
+- A datetime class that not only tracks timestamp but also their accuracy, \
+which simplifies dealing with timestamps originating from different sources
+- Simple process handling for spawning child processes with redirected I/O
+"
+HOMEPAGE="https://lib.filezilla-project.org"
+COPYRIGHT="2015-2016 Tim Kosse"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/filezilla/files/libfilezilla/$portVersion/libfilezilla-$portVersion.tar.bz2"
+CHECKSUM_SHA256="73c3ada6f9c5649abd93e6a3e7ecc6682d4f43248660b5506918eab76a7b901b"
+PATCHES="libfilezilla-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libfilezilla$secondaryArchSuffix = $portVersion compat >= 1
+	lib:libfilezilla$secondaryArchSuffix = 0.6.1 compat >= 0
```

Comment:
To find out what version should be declared for both the **`lib:libfilezilla$secondaryArchSuffix`** line (in PROVIDES) and the **`devel:libfilezilla$secondaryArchSuffix`** line (in PROVIDES_devel) you just need to check what x.y.z you get for develop/lib/libfilezilla.so.x.y.z when you list the contents of the devel package with «**`package list /path/to/libfilezilla_devel*.hpkg`**».